app/views/partials/personal_key/_entry_fields.html.slim

```diff
@@ -0,0 +1,8 @@
+.mb3
+  label.block.bold#personal-key-label
+    = t('simple_form.required.html') + t('forms.two_factor.recovery_code')
+  - Figaro.env.recovery_code_length.to_i.times do |_|
```

Comment:
same feedback as last time, can we drop the block argument here? or is that a slim thing? ex:

```ruby
times.do
```